# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../mine/ncurses.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

link against ncurses, not just curses. libcurses was not found.
missing compatibility link?

amery

--- ./Makefile.orig	2006-09-06 11:12:35.000000000 +0000
+++ ./Makefile	2006-09-06 11:12:46.000000000 +0000
@@ -89,7 +89,7 @@
 	$(CC) $(MINE_ALL_OBJS) -static -o mine.static
 
 gasgui: $(GAS_OBJ)
-	$(CC) $(GAS_OBJ) -ldialog -lcurses -lm -o gasgui
+	$(CC) $(GAS_OBJ) -ldialog -lncurses -lm -o gasgui
 
 install: all
 	install mine $(sysprefix)/bin/mine
